apps/editor/src/app/panel/memberPlanEditPanel.tsx

```diff
@@ -240,9 +240,9 @@ function MemberPlanEditPanel({id, onClose, onSave}: MemberPlanEditPanelProps) {
       </Drawer.Header>
       <Drawer.Body>
         <Panel>
-          <Form.Group controlId="memberPlanName">
```

Comment:
Fixes membership creation. Already fixed on master but needed it for testing things out